### DISCUSSION TITLE
An unexpected connection that should never have been allowed to connect.

### DISCUSSION THREAD

**Discussion by ninmba** [evilsocket/opensnitch#1221] [Show and tell]
Hello,
I'm running Debian Testing x64 + KDE. with OpenSnitch v1.6.6
OpenSnitch is working fine and blocking everything as expected. I can see the blocked apps in the Events tab.

But I noticed a connection that I didn't expect. How did this happen?
Firstly, I had a Konsole window open running the command:
watch 'netstat -plant'
At that time, the output was empty, meaning no apps were connected. (it can be with UDP, but that's not my subject).
However, after I left my PC for a few hours and came back, I saw several connections from Dolphin - KDE file manager, which were ESTABLISHED to IPs like 172.x.x.x and 177.x.x.x.  
I searched for the 172.x.x.x IP and found that it belongs to Akamai in the US. 
Unfortunately, I didn't save the information or take a screenshot of the Konsole window with the netstat output.
My settings in OpenSnitch are configured to deny all new apps by default, except the rules to 'Allow' some applications, such as web browsers, ping, and chat apps.
So my question is: How did OpenSnitch allow Dolphin to connect?

**Comment by gustavo-iniguez-goya** [comment]
Hi @ninmba ,

Probably the connections were allowed by some rule. You can go to the Applications tab  -> double click on the dolphin binary, and review what rule allowed those connections.
